lib/ddtrace/contrib/active_support/notifications/subscriber.rb

```diff
@@ -36,7 +36,13 @@ def on_subscribe(&block)
             # Should be triggered by the inheriting class.
             def subscribe!
               return subscribed? if subscribed? || on_subscribe_block.nil?
-              on_subscribe_block.call
+              begin
+                on_subscribe_block.call
+              rescue => e
+                puts e
```

Comment:
Was this left in here by accident after debugging?

Reply:
Ups, sorry for that. Removed! And thanks for spotting it. I'll check the diff next time, before resubmitting for review.